scripts/prepack.ts

```diff
@@ -11,9 +11,13 @@ import * as path from 'path';
 
 const NPM_BUILD_DIR = 'build/npm';
 const BUILD_DIR = 'build';
-const NPM_IGNORE = fs.existsSync('.npmignore') ? '.npmignore' : '../../.npmignore';
 
-const ASSETS = ['README.md', 'LICENSE', 'package.json', NPM_IGNORE] as const;
+const ASSETS = ['README.md', 'LICENSE', 'package.json'];
+
+if (fs.existsSync('.npmignore')) {
+  ASSETS.push('.npmignore');
+}
```

Comment:
l: I think we already check for existence further below when copying the assets. Do we need this check here?

Reply:
Down below we check for an asset, but we throw if it doesn't exist. I could switch the logic below to just copy if a file exists and noop otherwise.